.env.sample

```diff
@@ -9,6 +9,8 @@ BACKUPS_VOLUME_DRIVER=local
 C_FORCE_ROOT=1
 FORCE_REINIT=false
 INVOKE_LOG_STDOUT=true
+GEONODE_LOG_STDOUT=true
+CELERY_LOG_STDOUT=true
```

Comment:
I don't see these new `GEONODE_LOG_STDOUT` and `CELERY_LOG_STDOUT` vars even used.
Their purpose should have been to either send logs to a file or redirect to stdout.
For GeoNode this would been tweaking the [uwsgi logger](https://github.com/GeoNode/geonode-project/blob/7393bb71b14691d4ce135d84fd3c1b1b0c9c2eb5/src/uwsgi.ini#L4) 
For Celery it means working on [this line](https://github.com/GeoNode/geonode-project/blob/master/src/celery.sh#L2).

If it's to complicated we can leave this option out for the moment, and just implement the persistent logging into a volume.

Reply:
I propose to leave these `GEONODE_LOG_STDOUT` and `CELERY_LOG_STDOUT` options out for the moment.